tests/framework/common_apps.go

```diff
@@ -212,14 +213,17 @@ func (td *OsmTestData) GetPodsForLabel(ns string, labelSel metav1.LabelSelector)
 
 // SimplePodAppDef defines some parametrization to create a pod-based application from template
 type SimplePodAppDef struct {
-	Namespace   string
-	Name        string
-	Image       string
-	Command     []string
-	Args        []string
-	Ports       []int
-	AppProtocol string
-	OS          string
+	Namespace          string
+	PodName            string
+	ServiceName        string
+	ServiceAccountName string
```

Comment:
Thanks, much needed to avoid setting the same name for namespace, service, and service accounts.